app/service.py

```diff
@@ -45,56 +44,53 @@ async def shutdown():
     await db.disconnect()
 
 
-@app.post("/api/v1/providers", response_model=schemas.APIKey)
```

Comment:
Is there a reason to move these definitions down below? Ideally, we should just include the tags right? Let's minimize these changes.

Reply:
I moved the functions so that the first public endpoints, if we don't move them, are the `apiKeyMaster` first

| Before | After|
|-------|---------|
| ![Antes]( https://github.com/user-attachments/assets/0be81f2e-1731-4fb0-95ad-17d328efb6f5) | ![Después](https://github.com/user-attachments/assets/c359e375-ee81-4fc3-b6aa-fe123514a28f) |